pkg/config/config.go

```diff
@@ -270,6 +273,15 @@ func (c *Config) GetStepConfig(flagValues map[string]interface{}, paramJSON stri
 		}
 	}
 
+	// hooks need to have been loaded from the defaults before the server URL is known
+	err = c.setTrustEngineConfiguration(stepConfig.HookConfig)
+	if err != nil {
+		log.Entry().WithError(err).Debug("Trust Engine lookup skipped for all parameters due to missing or incorrect configuration")
```

Comment:
simplify, details should be in linked error

```suggestion
		log.Entry().WithError(err).Debug("Trust Engine lookup skipped due to missing or incorrect configuration")
```

Reply:
Done.